packages/app-project/src/screens/ClassifyPage/components/CollectionsModal/components/CreateCollection/CreateCollection.js

```diff
@@ -0,0 +1,81 @@
+import counterpart from 'counterpart'
+import { Box, Button, CheckBox, FormField, TextInput } from 'grommet'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function CreateCollection ({
+  collection,
+  disabled,
+  onChange,
+  onSubmit
+}) {
+  const checkbox = React.createRef()
+  const textInput = React.createRef()
+  const { display_name, private: isPrivate } = collection
+  function updateCollection () {
+    const display_name = textInput.current.value
+    const isPrivate = checkbox.current.checked
+    onChange({
+      display_name,
+      private: isPrivate
+    })
+  }
+  return (
+    <React.Fragment>
+      <FormField
+        htmlFor='collectionName'
+        label={counterpart('CreateCollection.label')}
+      >
+        <TextInput
+          id='collectionName'
+          onChange={updateCollection}
+          ref={textInput}
+          value={display_name}
+        />
+      </FormField>
+      <Box
+        align='center'
+        margin={{ top: 'medium' }}
+        pad={{ top: 'small' }}
+      >
+        <Button
```

Comment:
Should this have `type="submit"`?

Reply:
If this is a form, then yes. If not, then I don't think you can have submit buttons outside a form.